be/src/Project/Web/Seller/Seller.Web/Areas/Clients/Controllers/ClientNotificationTypeController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Foundation.ApiExtensions.Definitions;
+using Foundation.Extensions.ModelBuilders;
+using Foundation.PageContent.ComponentModels;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Mvc;
+using Seller.Web.Areas.Clients.ViewModels;
+using System;
+using System.Globalization;
+using System.Threading.Tasks;
+
+namespace Seller.Web.Areas.Clients.Controllers
+{
+    [Area("Clients")]
+    public class ClientNotificationTypeController : Controller
```

Comment:
**suggestion** Should be ClientNotificationTypesController instead of ClientNotificationTypeController

Reply:
This controller is for returning view to edit notificationType and it is correct name because it's mean one notificationType is editing, like ClientRoleController, ClientGroupController etc.